modules/caddyhttp/encode/encode.go

```diff
@@ -84,17 +84,49 @@ func (enc *Encode) Provision(ctx caddy.Context) error {
 
 	if enc.Matcher == nil {
 		// common text-based content types
+		// list based on https://developers.cloudflare.com/speed/optimization/content/brotli/content-compression/#compression-between-cloudflare-and-website-visitors
+		// with application/atom+rss added
 		enc.Matcher = &caddyhttp.ResponseMatcher{
 			Headers: http.Header{
 				"Content-Type": []string{
-					"text/*",
-					"application/json*",
-					"application/javascript*",
-					"application/xhtml+xml*",
 					"application/atom+xml*",
+					"application/eot*",
+					"application/font*",
+					"application/font-sfnt*",
+					"application/font-woff*",
```

Comment:
Isn't `application/font-sfnt*` and `application/font-woff*`  redundant because the `application/font*` will match for those as well?

Reply:
Good catch. I'll update.